SpatialGDK/Source/SpatialGDK/Public/Schema/ServerToServerRPCEndpointLegacy.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved	
```

Comment:
This is essentially duplicated boilerplate code from `ServerRPCEndpointLegacy.h`